pkgs/games/prismlauncher/default.nix

```
@@ -0,0 +1,94 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  jdk8,
+  jdk,
+  zlib,
+  file,
+  wrapQtAppsHook,
+  xorg,
+  libpulseaudio,
+  qtbase,
+  libGL,
+  quazip,
+  glfw,
+  openal,
+  extra-cmake-modules,
+  tomlplusplus,
+  ghc_filesystem,
+  msaClientID ? "",
+  jdks ? [jdk jdk8],
+}: let
+  rev = "243600b75babc636bdd1ac0dc8fd5fa4a2db1a0c";
+  sha256 = "sha256-cDwa6NeZew+MubtRmUViLA8gUaoH3AMxD9PdQcgSgSU=";
+
+  libnbtplusplus = fetchFromGitHub {
+    owner = "ljfa-ag";
+    repo = "libnbtplusplus";
+    rev = "v2.5";
+    sha256 = "sha256-4PHKPbWIWAy9ym25M4SnllTkrqnR+g/0AOqyMBZcdX4=";
+  };
+in
+  stdenv.mkDerivation {
+    pname = "prismlauncher";
+    version = builtins.substring 0 7 rev;
```

Current convention for builds from head with no version is unstable-date.
```suggestion
    version = "unstable-2022-10-18";
```